lib/src/component/dom_components.dart

```diff
@@ -35,7 +35,7 @@ DomProps domProps([Map backingMap]) => new DomProps(null, backingMap);
 
 typedef DomProps DomPropsFactory();
 
-class DomProps extends component_base.UiProps with DomPropsMixin, ReactPropsMixin {
+class DomProps extends component_base.UiProps with DomPropsMixin {
```

Comment:
This is part of "_Fix mixing in the same props class twice_"; `ReactPropsMixin` is already mixed in by `component_base.UiProps`